packages/api/src/controllers/webhook.ts

```diff
@@ -44,7 +44,7 @@ const fieldsMap: FieldsMap = {
   url: `webhook.data->>'url'`,
   blocking: `webhook.data->'blocking'`,
   deleted: `webhook.data->'deleted'`,
-  createdAt: `webhook.data->'createdAt'`,
+  createdAt: { val: `webhook.data->'createdAt'`, type: "int" },
```

Comment:
Yeah this should be fine. FTR another approach would be changing it to 
```
createdAt: `webhook.data->>'createdAt'`,
```
(`->>` instead of `->`)

Which would read the field as a string instead of jsonb. This would render inaccurate ordering of an int tho as it'd be comparing them as strings, but at least it would use the index that we have on the field (related to https://github.com/livepeer/livepeer-com/blob/16fd70915821dfbbc21bfd6bd8d3b65c90a1ade9/packages/api/src/store/stream-table.ts#L116).

I checked the other APIs we have though, and we are always doing this `type: "int"`, so let's leave it like this here for consistency. Once we fix these indexes we can fix all the places.